server/aws-lsp-codewhisperer/src/shared/streamingClientService.ts

```diff
@@ -74,6 +75,11 @@ export class StreamingClientServiceToken extends StreamingClientServiceBase {
             endpoint,
             token: tokenProvider,
             retryStrategy: new ConfiguredRetryStrategy(0, (attempt: number) => 500 + attempt ** 10),
+            requestHandler: {
+                keepAlive: true,
+                requestTimeout: clientTimeoutMs,
+                connectionTimeout: clientTimeoutMs,
```

Comment:
is this increasing or decreasing the timeout? that's the key part of the "Solution" description that's missing. "set streamingClient timeout config" is not a useful Solution description.

if you are increasing the timeout to ... 3 minutes, the question is why? will that prevent the client from retrying for up to 3 minutes, even when there is no activity on the connection?

how many retries are configured currently?

Reply:
removed `connectionTimeout` setting as it is not needed and can cause regression